common/src/main/java/com/amazonaws/services/schemaregistry/common/configs/GlueSchemaRegistryConfiguration.java

```diff
@@ -101,6 +104,9 @@ private void buildSchemaRegistryConfigs(Map<String, ?> configs) {
         validateAndSetAWSSourceEndpoint(configs);
         validateAndSetAWSTargetEndpoint(configs);
         validateAndSetRegistryName(configs);
+        validateAndSetSourceRegistryName(configs);
+        validateAndSetTargetRegistryName(configs);
+        validateAndSetReplicateSchemaVersionCount(configs);
```

Comment:
Can we limit these configurations to just the connector as they don't concern the core GSR library?

Reply:
Removed. Moved this to converter module